### PR TITLE
chore(readme): add slack community invite link

### PR DESCRIPTION
**Pull request by nellyk**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <img src='https://github.com/americanexpress/one-app/raw/master/one-app.png' alt="One App - One Amex" width='50%'/>
 </h1>
 
+[![Slack](https://img.shields.io/badge/slack-join%20chat-blue)](https://join.slack.com/t/one-amex/shared_invite/enQtOTA0MzEzODExODEwLTlmYzI1Y2U2ZDEwNWJjOTAxYTlmZTYzMjUyNzQyZTdmMWIwZGJmZDM2MDZmYzVjMDk5OWU4OGIwNjJjZWRhMjYV)
```
Comment (JAdshead) [inline README.md]: when i follow that url is comes up as "This invite link is no longer active."

Comment (anescobar1991) [reply README.md]: weird... It worked for me yesterday

Comment (nellyk) [reply README.md]: Thanks for checking @JAdshead  yeah the link wasn't working! i've updated it

**Comment by AlexsJones** [comment]
Why was this closed?

**Comment by jsmartfo** [comment]
@nellyk  👀

**Pull request closed without merging**